spec/models/event_spec.rb

```diff
@@ -1,14 +1,52 @@
 require 'rails_helper'
 
 RSpec.describe Event, type: :model do
-  before(:context) do
-  	@event = FactoryBot.create :event
+
+it "should have an attribute deadline" do
+    @event = FactoryBot.create :event
+    @date = Date.new(2017,11,16)
+    expect(@event.deadline).to eq(@date)
+  end
```

Comment:
aren't "should have attribute" tests the tests for validation? So should also check, that the attribute can't be nil.